geps/gep-2649/metadata.yaml

```
@@ -2,7 +2,7 @@ apiVersion: internal.gateway.networking.k8s.io/v1alpha1
 kind: GEPDetails
 number: 2649
 name: Inherited Policy Attachment
-status: Provisional
+status: Memorandum
```

```suggestion
status: Declined
```

And an `obsoletedBy` section pointing to GEP-713.